specifyweb/frontend/js_src/lib/components/FormCommands/index.tsx

```diff
@@ -98,7 +99,9 @@ const commandRenderers: {
               header={label}
               onClose={handleHide}
             >
-              {interactionsText.preparationsCanNotBeReturned()}
+              {interactionsText.preparationsCanNotBeReturned({
+                preparationTable: String(tables.Preparation.label).toLowerCase(),
```

Comment:
String() is probably not needed here and in the following usages

```suggestion
                preparationTable: tables.Preparation.label.toLowerCase(),
```